configs/razor_subnets/README.md

```diff
@@ -0,0 +1,54 @@
+# Projecs Based on MMRAZOR
```

Comment:
Considering other subsequent mmrazor related tasks, this configuration folder should be named `configs/razor/subnets`